.changeset/violet-flowers-poke.md

```diff
@@ -0,0 +1,45 @@
+---
+"@latticexyz/store": patch
```

Comment:
```suggestion
```
there's no real user-facing store changes, just conditionally defaulting `indexFilename` to `false` if `namespaces` is used, so would probably opt to exclude this changeset from this description

I could see leaving this here if store had `namespaces` support too, but we haven't migrated that yet